From b3a5e7963258f7615e7ce5f28314bb9f79d98936 Mon Sep 17 00:00:00 2001
From: GvantsaKiknadzee <gvantsa.kiknadze17@gmail.com>
Date: Wed, 15 May 2024 14:28:56 +0400
Subject: [PATCH] changed 2 lines in firstFile.txt

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index d3d1712..70f6398 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -6,5 +6,5 @@
 1
 1
 1
-1
-1
+modify
+modify1
-- 
2.38.0

